vendor/src/github.com/microsoft/hcsshim/layerutils.go

```diff
@@ -84,9 +85,14 @@ func layerPathsToDescriptors(parentLayerPaths []string) ([]WC_LAYER_DESCRIPTOR,
 	var layers []WC_LAYER_DESCRIPTOR
 
 	for i := 0; i < len(parentLayerPaths); i++ {
-		// Create a layer descriptor, using the folder path
+		// Create a layer descriptor, using the folder name
```

Comment:
Ok, am I able to test this in a windows build or not? (Although I still don't have a working windows build :( )